content/c/concepts/structures/structures.md

```diff
@@ -0,0 +1,132 @@
+---
+Title: 'Structures'
+Description: 'A structure is a collection that can be used to group different types of data together.'
```

Comment:
Is there a way to phrase this without using the word "collection", it might invite confusion with collections as they appear in other languages.

Reply:
@SSwiniarski I was using the terminology from the [Codecademy course on c structures](https://www.codecademy.com/courses/learn-c/lessons/c-structures/exercises/introduction). How about just taking it out instead?

```suggestion
Description: 'A structure can be used to group different types of data together.'
```